types/react-loadable/react-loadable-tests.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import Loadable, { LoadingComponentProps } from 'react-loadable';
+
+class LoadingComponent extends React.Component<LoadingComponentProps, {}> {
+  render() {
+    return <div>{this.props.isLoading}</div>;
+  }
+}
+
+interface ComponentProps {
+  text: string;
+}
+
+const Component: React.SFC<ComponentProps> = ({ text }) => <div>{text}</div>;
+
+const Loadable100 = Loadable({
+  // a module shape with 'export = Component' / 'module.exports = Component'
+  loader: () => Promise.resolve(Component),
```

Comment:
The `Component` and the various shapes can perhaps be split into separate test-specific files once dynamic import support is in typescript, for writing this test how it's intended to be used -- `() => import('./Component')`.